### DISCUSSION TITLE
error 500 on my apps page, need help

### DISCUSSION THREAD

**Discussion by sakethitsme** [vercel/community#3646] [Help]
### Summary

I have created a small html page with proxy.js file, which already has 'const fetch = require('node-fetch');' in it.

and when i try to run the page, this is what i am getting in the vercel logs, can anyone help, where I am doing wrong or do I need to change anything?

"Cannot find module 'node-fetch'
Require stack:
- /var/task/api/proxy.js
Did you forget to add it to "dependencies" in `package.json`?
RequestId: 52c03d45-6ed4-480f-a2ac-4e2a3ddf427e Error: Runtime exited with error: exit status 1
Runtime.ExitError"

### Steps to Reproduce

access https://monitoryourapp.vercel.app/,

enter your desired url and enter time interval and click on check status, it will show the error

**Comment by cggaurav** [comment]
Have a similar issue with `axios`, I think the reinstall of package.json isn't happenning despite not using the build cache and force deploying.

**Reply by cggaurav** [reply]
Likely related to https://vercel.com/docs/deployments/configure-a-build#corepack
